dipper/sources/EBIGene2Phen.py

```diff
@@ -0,0 +1,315 @@
+import csv
+import gzip
+import requests
+from contextlib import closing
+from typing import List, Optional
+
+from dipper.sources.Source import Source
+from dipper.models.Dataset import Dataset
+from dipper.models.assoc.G2PAssoc import G2PAssoc
+from dipper.models.Model import Model
+from dipper.models.Genotype import Genotype
+
+
+logger = logging.getLogger(__name__)
+
+
+class EBIGene2Phen(Source):
+    """
+    From EBI:
+    The gene2phenotype dataset (G2P) integrates data on genes,
+    variants and phenotypes for example relating to developmental
+    disorders. It is constructed entirely from published literature,
+    and is primarily an inclusion list to allow targeted filtering of
+    genome-wide data for diagnostic purposes. The dataset was compiled
+    with respect to published genes, and annotated with types of disease-
+    causing gene variants. Each row of the dataset associates a gene with
+    a disease phenotype via an evidence level, inheritance mechanism and
+    mutation consequence. Some genes therefore appear in the database
+    more than once, where different genetic mechanisms result in
+    different phenotypes.
+
+    Disclaimer: https://www.ebi.ac.uk/gene2phenotype/disclaimer
+    Terms of Use: https://www.ebi.ac.uk/about/terms-of-use#general
+    Documentation: https://www.ebi.ac.uk/gene2phenotype/documentation
+                   https://www.clinicalgenome.org/site/assets/files/
+                   2757/fitzpatrick_ddg2p.pdf
+
+    This script operates on the Developmental Disorders (DDG2P.csv) file
+    In the future we may update to include the cancer gene disease pairs
+    in the  CancerG2P.csv file
+    """
+
+    EBI_BASE = "https://www.ebi.ac.uk/gene2phenotype/downloads/"
+
+    files = {
+        'developmental_disorders': {
+            'file': 'DDG2P.csv.gz',
+            'url': EBI_BASE + 'DDG2P.csv.gz'}
+    }
+
+    map_files = {
+        # map for unmapped disease labels to MONDO ids
+        'mondo_map': 'https://data.monarchinitiative.org/dipper/'
+                     'cache/unmapped_ebi_diseases.tsv'
+    }
+
+    def __init__(self, graph_type, are_bnodes_skolemized):
+        super().__init__(graph_type, are_bnodes_skolemized, 'ebi_g2p')
+
+        self.dataset = Dataset(
+            'ebi',
+            'EBI',
+            'https://www.ebi.ac.uk/gene2phenotype',
+            None,
+            None,
+            'https://www.ebi.ac.uk/about/terms-of-use#general',
+        )
+
+        # Load mondo map file
+        self.mondo_map = {}
+        map_url = self.map_files['mondo_map']
+        with closing(requests.get(map_url, stream=True)) as map_fh:
+            for line in map_fh.iter_lines():
+                row = line.decode('utf-8').split('\t')
+                self.mondo_map[row[0]] = row[1]
+
+        self.global_terms = self.open_and_parse_yaml('../../translationtable/global_terms.yaml')
+        self.translation_table = self.open_and_parse_yaml('../../translationtable/ebi.yaml')
```

Comment:
Going to ask the filename be the same base that is used elsewhere   
which is the ingest file, name lowercased,  sans .py 

`ebigene2phen.yaml` here